packages/runtime/container-runtime/src/test/blobManager.spec.ts

```diff
@@ -172,15 +173,23 @@ export class MockRuntime
 		this.ops = [];
 	}
 
-	public async processBlobs(resolve: boolean, canRetry?: boolean, retryAfterSeconds?: number) {
+	public async processBlobs(resolve: boolean, canRetry: boolean = false, retryAfterMs?: number) {
 		const blobPs = this.blobPs;
 		this.blobPs = [];
 		if (resolve) {
 			this.processBlobsP.resolve();
 		} else {
-			const err: any = new Error("fake error");
-			err.canRetry = canRetry;
-			err.retryAfterSeconds = retryAfterSeconds;
+			const err = createGenericNetworkError(
+				"fake driver error",
+				// TODO: https://dev.azure.com/fluidframework/internal/_workitems/edit/5913
+				{ canRetry, retryAfterMs },
+				{
+					// To avoid using the default retry time added when we don't have retryAfterMs
```

Comment:
I created an item to track a but I saw here where we are losing the retryAfterMs property when creating a GenericNetworkError